library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -852,8 +924,15 @@ class SwipeActionView : FrameLayout {
         } else {
             leftSwipeRipple.restart()
         }
+        if(isHalfwaySwipe) {
```

Comment:
This should be done inside animateContainer on line 858 instead of regular onSwipedRight/onSwipedLeft callback.

Reply:
Calls to `onSwipedHalfwayRight/onSwipedHalfwayLeft` should replace calls to `onSwipedRight/onSwipedLeft` if it is a halfway swipe. In this function (line 858/935 to 862/939). Right now both of them will be called